apps/angular/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,19 @@
 import { NgFor } from '@angular/common';
 import { Component } from '@angular/core';
+import { ArrayToTextPipe } from './array-to-text.pipe';
 
 @Component({
   standalone: true,
-  imports: [NgFor],
+  imports: [NgFor, ArrayToTextPipe],
   selector: 'app-root',
   template: `
-    <div *ngFor="let person of persons; let index = index">
-      {{ heavyComputation(person, index) }}
+  @for (person of persons; let index = $index ;track person) {
+    <div>
+      {{ person | arrayToText: index }}
```

Comment:
That's perfect, you can even call `$index` immediately without creating an alias

Reply:
Oh that's nice. Thanks buddy